main.c

```diff
@@ -107,6 +107,11 @@
 #include "shared-bindings/wifi/__init__.h"
 #endif
 
+#ifdef CIRCUITPY_BOOT_COUNTER
```

Comment:
In `py/circuitpy_mpconfig.h` put an `ifndef` that defines this as 0. That way we ensure that the correct files were included to have this defined.

```suggestion
#if CIRCUITPY_BOOT_COUNTER
```